src/swap/hooks/useFromTo.test.ts

```diff
@@ -57,6 +61,77 @@ describe('useFromTo', () => {
       loading: false,
       setLoading: expect.any(Function),
       error: null,
+      refetch: expect.any(Function),
     });
   });
+
+  it('should call fromTokenResponse.refetch when from.refetch is called', async () => {
+    const mockFromRefetch = vi.fn().mockResolvedValue(undefined);
+    const mockToRefetch = vi.fn().mockResolvedValue(undefined);
+
+    (useSwapBalances as vi.Mock).mockReturnValue({
+      fromTokenResponse: { refetch: mockFromRefetch },
+      toTokenResponse: { refetch: mockToRefetch },
+    });
+
```

Comment:
Let's get into the habit to remove spaces from tests, so they are much easier to read, review, mantain.